dotcom-rendering/src/components/Masthead/Titlepiece/Titlepiece.tsx

```diff
@@ -124,7 +124,7 @@ const pillarsNavStyles = css`
 `;
 
 const subNavStyles = css`
-	grid-column: content-start / content-end;
+	${gridFullWidth}
```

Comment:
Looks like this just ends up being `grid-column: content-start / main-column-end;` - could we specify that directly here?